winml/lib/Api.Ort/OnnxruntimeModel.cpp

```diff
@@ -148,8 +148,28 @@ STDMETHODIMP ModelInfo::GetVersion(int64_t* out) {
   return S_OK;
 }
 
+struct CaseInsensitiveHash {
```

Comment:
do you need to write your own hash? you're just trying to override the equality part, right? can you use "class Hash = hash<Key>" (see http://www.cplusplus.com/reference/unordered_map/unordered_map/).

Reply:
I think you do... else strings that should compare the same will have different hashes